documentation/src/main/asciidoc/titles/cli/cli-docinfo.html

```diff
@@ -1,6 +1,9 @@
 <link rel="canonical" href="https://infinispan.org/docs/stable/titles/cli/cli.html">
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jstree/3.3.1/themes/default/style.min.css">
 <link rel="stylesheet" href="../../css/css.css">
+<link rel="stylesheet" href="/assets/css/clipboard.css">
```

Comment:
are you sure this is working as expected? `assets` directory does not exist

Reply:
Hey Pedro, yeah it seems a little wonky but that href is for the community site only. 

https://infinispan.org/assets/css/clipboard.css

This does mean that you don't get the copy to clipboard function when you build locally with maven but I think that's acceptable.

Reply:
Also if you want to see the copy to clipboard in action it's already there for the Operator docs e.g.: https://infinispan.org/docs/infinispan-operator/2.2.x/operator.html#infinispan-cr_creating-clusters

Reply:
ah that explains why I couldn't check anything locally :rofl: